<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Cormorant+Garamond:ital,wght@0,300..700;1,300..700&family=Noto+Serif:ital,wght@0,100..900;1,100..900&family=VT323&display=swap" 
    rel="stylesheet">
    <link href="https://fonts.googleapis.com/css2?family=MonteCarlo&display=swap" 
    rel="stylesheet">
    <link href="https://fonts.googleapis.com/css2?family=Oranienbaum&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="../../pinkAndYellow.css">
    <link rel="icon" type="image/png" href="../../favicon.png">
    <title>Model: Green Team</title>
</head>
<body>
  <div id="loader">
    <video autoplay muted loop playsinline preload="auto">
      <source src="../../load.mov" type="video/mp4">
    </video>
    <h1>LOADING...</h1>
  </div>
    <nav>
        <div class="logo"><a href="../../index.html">Green Team</a></div>
        <div class="nav-items">

          <div class="dropdown">
            <button class="dropbtn" id="projectBtn">Project</button>
            <div class="dropdown-content" id="projectMenu">
              <a href="../project/description.html">Description</a>
              <a href="../project/design.html">Design</a>
              <a href="../project/proofOfConcept.html">Proof of Concept</a>
              <a href="../project/engineering.html">Engineering</a>
              <a href="../project/results.html">Results</a>
              <a href="../project/implementation.html">Implementation</a>
              <a href="../project/safety.html">Safety</a>
              <a href="../project/contribution.html">Contribution</a>
            </div>
          </div>

          <div class="dropdown">
            <button class="dropbtn" id="wetLabBtn">Wet Lab</button>
            <div class="dropdown-content" id="wetlabMenu">
              <a href="../wetlab/experiments.html">Experiments</a>
              <a href="../wetlab/notebook.html">Notebook</a>
              <a href="../wetlab/measurements.html">Measurement</a>
              <a href="../wetlab/labSafety.html">Lab Safety</a>
            </div>
          </div>

          <div class="dropdown">
            <button class="dropbtn" id="dryLabBtn">Dry Lab</button>
            <div class="dropdown-content" id="drylabMenu">
              <a href="model.html">Model</a>
              <a href="parts.html">Parts</a>
              <a href="software.html">Software</a>
            </div>
          </div>

          <div class="dropdown">
            <button class="dropbtn" id="engagementBtn">Engagement</button>
            <div class="dropdown-content" id="engagementMenu">
              <a href="../engagement/hp.html">Human Practices</a>
              <a href="../engagement/ihp.html">Integrated Human Practices</a>
              <a href="../engagement/collaborations.html">Collaborations</a>
              <a href="../engagement/sustainability.html">Sustainability</a>
              <a href="../engagement/education.html">Education</a>
            </div>
          </div>

          <div class="dropdown">
            <button class="dropbtn" id="teamBtn">Team</button>
            <div class="dropdown-content" id="teamMenu">
              <a href="../team/members.html">Members</a>
              <a href="../team/attributions.html">Attributions</a>
              <a href="../team/medals.html">Medals</a>
              <a href="../team/sponsors.html">Sponsors</a>
            </div>
          </div>

        </div>
      </nav>
      <div class="head fade-in">
        <h1 class="title">Modelling</h1>
      </div>
  <div class="page">
    <aside id="toc-container">
      <button id="toc-toggle">Contents</button>
      <nav id="toc"></nav>
    </aside>

    <main id="content">
      <h2>Whole-Genome BLAST Analysis Against the Target Aldehyde Gene</h2>
      <div class="dividerbar"></div>
      <p>We performed a whole-genome BLAST analysis of <em>Pseudomonas putida F1</em> against our target 
        aldehyde-related gene (cymC) to identify homologous sequences and confirm the presence.
        <br><br>
        Before designing a gene knockout strategy, it was essential to:</p>
        <ul>
          <li>Confirm that cymC is present as a single, well-defined locus in the genome</li>
          <li>Check for paralogous or homologous genes that could compensate for cymC deletion</li>
          <li>Ensure that targeting cymC would not unintentionally affect other aldehyde-metabolising enzymes</li>
          <li>Verify that the gene sequence used for the Upstream Homologous Arm and the Downstream Homologous Arm design is accurate</li>
        </ul>

        <p>A BLASTn search was performed using the <em>Pseudomonas putida F1</em> genome as the subject.</p>
        <div class="image-and-caption">
        <img src="m1.png">
        <p class="caption"></p>
      </div>

      <p>Inference : BLAST analysis showed a perfect match between the query gene and the <em>Pseudomonas putida F1</em> genome, with 100% identity 
        and an E-value of 0, indicating a highly significant alignment. </p>

      <h2>Aldehyde Dehydrogenase Specificity Analysis</h2>
      <div class="dividerbar"></div>
      <p>We performed a comparative BLAST analysis to evaluate whether other aldehyde dehydrogenase 
        enzymes present in <em>Pseudomonas putida F1</em> could potentially interfere with or compensate for 
        the function of our target aldehyde-related gene (cymC).A. putida contains multiple aldehyde 
        dehydrogenases involved in diverse metabolic pathways. <br>Before proceeding with a gene knockout 
        strategy, it was important to assess:</p>
        <ul>
          <li>Potential functional redundancy</li>
          <li>Possible off-target aldehyde metabolism</li>
          <li>The likelihood of other enzymes compensating for the loss of cymC</li>
        </ul>

        <p>Reviewed aldehyde dehydrogenase proteins from P. putida F1 were retrieved from UniProt (Swiss-Prot). 
          A total of six aldehyde dehydrogenase candidates were identified. The protein sequences were individually 
          subjected to BLAST analysis with CymC. </p>

      <div class="image-and-caption">
        <img src="m2.png">
        <p class="caption"></p>
      </div>

      <p>The BLAST results show that while P. putida has other aldehyde dehydrogenases, the closest hits (betB and astD) 
        have only moderate to low sequence similarity to cymC. This suggests they are unlikely to compensate for its 
        function, confirming cymC as a specific and suitable knockout target.</p>
     
      <h2>Identification of Upstream and Downstream Homology Arms</h2>
      <div class="dividerbar"></div>
      <p>We identified the regions flanking the target gene (cymC) in the <em>P. putida F1</em> genome to design the 
        upstream and downstream homology arms (UHA and DHA) required for homologous recombination.
      <br><br>
        Using the BLAST results from the genome vs. cymC gene, we determined the exact genomic coordinates of the 
        target gene.
        <br><br>     
        The orientation (plus/minus strand) was noted to correctly identify upstream and downstream regions.
        <br><br>
        Sequences upstream (UHA) and downstream (DHA) of the gene were extracted. Reverse complement sequences 
        were converted to 5′ → 3′ orientation to prepare them for primer design and cloning.
      </p> 
        <div class="image-and-caption">
        <img src="m3.jpeg">
      </div>

      <div class="image-and-caption">
        <img src="m4.jpeg">
      </div>
      
      <p>UHA sequence : <br></p>
      <p class="dna">5' - CTGCCATGAGACTGAAAGACAAAGTCGCAATCGTGACTGGTGCGGCAACCGGGA
        TTGGCAACGCTATCGTGCGCTCTTACCTTGCTGAGGGTGCGAAGGTGGTCATTGCTGATGTGAAAGGTGCTG
        AGGCAGCCGCGGCGGAACTGGGTGAAGACCTAGCGTTGGGTGTCTTTGCCGACGTTTCGGATCCTGATAGCA
        CCAAGCAGATGGCAAAGGCAGCACTGGATCGCTTCGGCAAGATCGACGTGCTCGTCAACAATGCCGGTATCT
        TCACCGGTCTCAACTATGTCCCGATGGAATCGATCTCGGTTGCGGATTGGGACAAGCTCTACAGTGTCAATG
        TCAAGGGGCCCTGGCTGTGCGCCAGTGCGGTGTCGGCCGCGATGCGCGAGGGTGGTGGTGGCAAGATCATCA
        ACATCGCCTCAGTAATCGCTCACATCGGAGCGCCCTTCATGTTGCACTACGTTTCCAGCAAGGGCGCTGTTG
        CGGCGATGACACGTGCCATGGCGCGCGAATTTGCGACGACCAAGGCAGGGATCAGCGTCAACTCCATCTCTC
        CCGGCTACACCCATTCGGAAAATGCGCTCGCAAACGCTCAGCAGCATGAGCAGTTCGAGGGCGTTTCTGCGT
        CCATGCGGGCAATCGATCGACCGCAAGTGCCGGCAGATATTGCGGGAGTGGCGCTTTGGTTGGCTAGCGACG
        AAGCCTCCTACGTAAACGGGCAGAACATTGTGGTGGATGGCGGCATTTACATGAGCCTCTGAGCGTCATTCG
        CCACGACATTCATCCTTAGTTGTGATTT – 3' </p>

      <p>DHA sequence : <br></p>
      <p class="dna">5’ - ACGAAATCCTGCCCTCGCGTGCGAGGACGTCACGTCCGCTTAAGGAGACCGATC
        <strong>ATG</strong>TGGGAATACATCAAGTACTACTTCGCACCCCTGGTACAAGTCTTTGCGATTC
        TGGGGTTCTATTGGGGTGGTCATTACACTTGGATTGCAATTGGGGCTTTTCCGGCCATTGCCATCCTTGACA
        TGCTGCTGCCGCTCGATCTATCTGAGCGCCGAATGAAGAACCATTTCTGGGCTTACTTGCCGATCTGGGCTTC
        GACGCTGTTATTGCCGTTCATGTACTTCGGCTTCGCGTGGTCGGTAGCAAATAACGATCTGACCGGATTGCAA
        ATGGCAGCCGGTGTGCTGGGGTTGGCATGGCTTTCGGTTGTGCCTGGCGTTCCTGCAACCCACGAGTTGTATC
        ACTCGCGCGGGCGGTTGGCTCGCTTTGTTGGGCGCTATGGCCAGATCGTATTTCTCGATGTCATGCGCATGGA
        AGTGCACGTAGTCGGCCACCACCGCGACGTGGGGACGGCTGAGGACATTGATACTGCGGCACGCGGTATGAAT
        CTGTACAGATTCGTCGTTCGTGCAGTGGTGGAGTCCGTCCGCTGGGAGCTGAAGCTGGATGCAGACAATCTGG
        AGAAACGCGGTTACGGTCGCTATTCGATCCGTCACTCGGTCTGGCGCGCGCTGCTGGCCATTGTAGTGTTCCT
        GGGGTGCATCTATGCCATTGGCGGTGGGATGGCGGTGGGTCTTTGTTCGATCTCGATGATCATCGCTCGTTTT
        TGGGTTGAGGCATTCAACTACTACCAGCATTACGGCCAAGTCCGTTTGGTCGGCATGCCCATCGAAAAGCGGC
        ACGTCTGGAATCACTTTGGAACGCTCTCACGCTTGT – 3'</p>

      <h2>Construction of the Insert DNA</h2>
      <div class="dividerbar"></div>
      <p>UHA and DHA sequences were joined in silico to form a continuous segment.
          <br><br>
        Restriction sites for XmaI and PstI were added at the ends to allow the insert to ligate with the plasmid backbone.
          <br><br>
        Primers were designed specific to this segment with the following parameters:</p>
        <ul>
          <li>Length: ~35 nucleotides</li>
          <li>GC content: 50–60%</li>
          <li>Melting temperature (T<sub>m</sub>)</li> 
        </ul>

        <div class="image-and-caption">
        <img src="m5.png">
        <p class="caption">Fig : pst1 Restriction Site</p>
        </div>

        <div class="image-and-caption">
        <img src="m6.png">
        <p class="caption">Fig : Xma1 Restriction Site</p>
        </div>

        <div class="image-and-caption">
        <img src="m7.png">
        <p class="caption">Forward primer design : CCCGGGCTGCCATGAGACTGAAAGA</p>
        </div>

        <div class="image-and-caption">
        <img src="m8.png">
        <p class="caption">Reverse Primer Design : GGAACGCTCTCACGCTTGTCTGCAG</p>
        </div>

      <h2>References</h2>
      <div class="dividerbar"></div>
      <ol>
        <li><a href="https://en.vectorbuilder.com/tool/gc-content-calculator.html">GC Content Calculator | VectorBuilder</a></li>
        <li><a href="https://www.neb.com/en/products/r0140-psti?srsltid=AfmBOop5h8nbatBPoXMHjdzSdOCw9BNa9uk0VPAhNIFVu2yQe9EW8m5w">PstI | NEB</a></li>
        <li><a href="https://www.neb.com/en/products/r0180-xmai?srsltid=AfmBOorQmYOQpW5N06osUuJjbh4YaOueoCRlOqLGk3Gaxz1slQKCeJMg">XmaI | NEB</a></li>
      </ol>
    </main>
  </div>
    <script src="../../script.js"></script>
</body>
<footer>
    <img src="../../footerpic.png" alt="why not">
    <p>© 2025 Green Team <br>
       <em>for the taskphase of</em> <br>
       Manipal BioMachines. <br>
       <br>
      View our <a href="https://github.com/poseidon-boi/mbm-taskphase-green-team"><em>source code</em></a>
    </p>
</footer>
</html>