<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Cormorant+Garamond:ital,wght@0,300..700;1,300..700&family=Noto+Serif:ital,wght@0,100..900;1,100..900&family=VT323&display=swap" 
    rel="stylesheet">
    <link href="https://fonts.googleapis.com/css2?family=MonteCarlo&display=swap" 
    rel="stylesheet">
    <link href="https://fonts.googleapis.com/css2?family=Oranienbaum&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="../../pinkAndYellow.css">
    <link rel="icon" type="image/png" href="../../favicon.png">
    <title>Model: Green Team</title>
</head>
<body>
  <div id="loader">
    <video autoplay muted loop playsinline preload="auto">
      <source src="../../load.mov" type="video/mp4">
    </video>
    <h1>LOADING...</h1>
  </div>
    <nav>
        <div class="logo"><a href="../../index.html">green team</a></div>
        <div class="nav-items">

          <div class="dropdown">
            <button class="dropbtn" id="projectBtn">Project</button>
            <div class="dropdown-content" id="projectMenu">
              <a href="../project/description.html">Description</a>
              <a href="../project/design.html">Design</a>
              <a href="../project/proofOfConcept.html">Proof of Concept</a>
              <a href="../project/engineering.html">Engineering</a>
              <a href="../project/results.html">Results</a>
              <a href="../project/implementation.html">Implementation</a>
              <a href="../project/safety.html">Safety</a>
              <a href="../project/contribution.html">Contribution</a>
            </div>
          </div>

          <div class="dropdown">
            <button class="dropbtn" id="wetLabBtn">Wet Lab</button>
            <div class="dropdown-content" id="wetlabMenu">
              <a href="../wetlab/experiments.html">Experiments</a>
              <a href="../wetlab/notebook.html">Notebook</a>
              <a href="../wetlab/measurements.html">Measurement</a>
              <a href="../wetlab/labSafety.html">Lab Safety</a>
            </div>
          </div>

          <div class="dropdown">
            <button class="dropbtn" id="dryLabBtn">Dry Lab</button>
            <div class="dropdown-content" id="drylabMenu">
              <a href="model.html">Model</a>
              <a href="parts.html">Parts</a>
              <a href="software.html">Software</a>
            </div>
          </div>

          <div class="dropdown">
            <button class="dropbtn" id="engagementBtn">Engagement</button>
            <div class="dropdown-content" id="engagementMenu">
              <a href="../engagement/hp.html">Human Practices</a>
              <a href="../engagement/ihp.html">Integrated Human Practices</a>
              <a href="../engagement/collaborations.html">Collaborations</a>
              <a href="../engagement/sustainability.html">Sustainability</a>
              <a href="../engagement/education.html">Education</a>
            </div>
          </div>

          <div class="dropdown">
            <button class="dropbtn" id="teamBtn">Team</button>
            <div class="dropdown-content" id="teamMenu">
              <a href="../team/members.html">Members</a>
              <a href="../team/attributions.html">Attributions</a>
              <a href="../team/medals.html">Medals</a>
              <a href="../team/sponsors.html">Sponsors</a>
            </div>
          </div>

        </div>
      </nav>
      <div class="head fade-in">
        <h1 class="title">Modelling</h1>
      </div>
  <div class="page">
    <aside id="toc-container">
      <button id="toc-toggle">Contents</button>
      <nav id="toc"></nav>
    </aside>

    <main id="content">
      <h2>Whole-Genome BLAST Analysis Against the Target Aldehyde Gene</h2>
      <div class="dividerbar"></div>
      <p>We performed a whole-genome BLAST analysis of <em>Pseudomonas putida F1</em> against our target 
        aldehyde-related gene (cymC) to identify homologous sequences and confirm the presence.
        <br><br>
        Before designing a gene knockout strategy, it was essential to:</p>
        <ul>
          <li>Confirm that cymC is present as a single, well-defined locus in the genome</li>
          <li>Check for paralogous or homologous genes that could compensate for cymC deletion</li>
          <li>Ensure that targeting cymC would not unintentionally affect other aldehyde-metabolising enzymes</li>
          <li>Verify that the gene sequence used for the Upstream Homologous Arm and the Downstream Homologous Arm design is accurate</li>
        </ul>

        <p>A BLASTn search was performed using the <em>Pseudomonas putida F1</em> genome as the subject.</p>
        <div class="image-and-caption">
        <img src="m1.png">
        <p class="caption"></p>
      </div>

      <p>Inference : BLAST analysis showed a perfect match between the query gene and the <em>Pseudomonas putida F1</em> genome, with 100% identity 
        and an E-value of 0, indicating a highly significant alignment. </p>

      <h2>Aldehyde Dehydrogenase Specificity Analysis</h2>
      <div class="dividerbar"></div>
      <p>We performed a comparative BLAST analysis to evaluate whether other aldehyde dehydrogenase 
        enzymes present in <em>Pseudomonas putida F1</em> could potentially interfere with or compensate for 
        the function of our target aldehyde-related gene (cymC).A. putida contains multiple aldehyde 
        dehydrogenases involved in diverse metabolic pathways. <br>Before proceeding with a gene knockout 
        strategy, it was important to assess:</p>
        <ul>
          <li>Potential functional redundancy</li>
          <li>Possible off-target aldehyde metabolism</li>
          <li>The likelihood of other enzymes compensating for the loss of cymC</li>
        </ul>

        <p>Reviewed aldehyde dehydrogenase proteins from P. putida F1 were retrieved from UniProt (Swiss-Prot). 
          A total of six aldehyde dehydrogenase candidates were identified. The protein sequences were individually 
          subjected to BLAST analysis with CymC. </p>

      <div class="image-and-caption">
        <img src="m2.png">
        <p class="caption"></p>
      </div>

      <p>The BLAST results show that while P. putida has other aldehyde dehydrogenases, the closest hits (betB and astD) 
        have only moderate to low sequence similarity to cymC. This suggests they are unlikely to compensate for its 
        function, confirming cymC as a specific and suitable knockout target.</p>
     
      <h2>Identification of Upstream and Downstream Homology Arms</h2>
      <div class="dividerbar"></div>
      <p>We identified the regions flanking the target gene (cymC) in the <em>P. putida F1</em> genome to design the 
        upstream and downstream homology arms (UHA and DHA) required for homologous recombination.
      <br><br>
        Using the BLAST results from the genome vs. cymC gene, we determined the exact genomic coordinates of the 
        target gene.
        <br><br>     
        The orientation (plus/minus strand) was noted to correctly identify upstream and downstream regions.
        <br><br>
        Sequences upstream (UHA) and downstream (DHA) of the gene were extracted. Reverse complement sequences 
        were converted to 5′ → 3′ orientation to prepare them for primer design and cloning.
      </p> 
        <div class="image-and-caption">
        <img src="m3.jpeg">
      </div>

      <div class="image-and-caption">
        <img src="m4.jpeg">
      </div>
      
      <p>UHA sequence : <br></p>
      <p class="dna">5' - CTGCCATGAGACTGAAAGACAAAGTCGCAATCGTGACTGGTGCGGCAACCGGGA
        TTGGCAACGCTATCGTGCGCTCTTACCTTGCTGAGGGTGCGAAGGTGGTCATTGCTGATGTGAAAGGTGCTG
        AGGCAGCCGCGGCGGAACTGGGTGAAGACCTAGCGTTGGGTGTCTTTGCCGACGTTTCGGATCCTGATAGCA
        CCAAGCAGATGGCAAAGGCAGCACTGGATCGCTTCGGCAAGATCGACGTGCTCGTCAACAATGCCGGTATCT
        TCACCGGTCTCAACTATGTCCCGATGGAATCGATCTCGGTTGCGGATTGGGACAAGCTCTACAGTGTCAATG
        TCAAGGGGCCCTGGCTGTGCGCCAGTGCGGTGTCGGCCGCGATGCGCGAGGGTGGTGGTGGCAAGATCATCA
        ACATCGCCTCAGTAATCGCTCACATCGGAGCGCCCTTCATGTTGCACTACGTTTCCAGCAAGGGCGCTGTTG
        CGGCGATGACACGTGCCATGGCGCGCGAATTTGCGACGACCAAGGCAGGGATCAGCGTCAACTCCATCTCTC
        CCGGCTACACCCATTCGGAAAATGCGCTCGCAAACGCTCAGCAGCATGAGCAGTTCGAGGGCGTTTCTGCGT
        CCATGCGGGCAATCGATCGACCGCAAGTGCCGGCAGATATTGCGGGAGTGGCGCTTTGGTTGGCTAGCGACG
        AAGCCTCCTACGTAAACGGGCAGAACATTGTGGTGGATGGCGGCATTTACATGAGCCTCTGAGCGTCATTCG
        CCACGACATTCATCCTTAGTTGTGATTT – 3' </p>

      <p>DHA sequence : <br></p>
      <p class="dna">5’ - ACGAAATCCTGCCCTCGCGTGCGAGGACGTCACGTCCGCTTAAGGAGACCGATC
        <strong>ATG</strong>TGGGAATACATCAAGTACTACTTCGCACCCCTGGTACAAGTCTTTGCGATTC
        TGGGGTTCTATTGGGGTGGTCATTACACTTGGATTGCAATTGGGGCTTTTCCGGCCATTGCCATCCTTGACA
        TGCTGCTGCCGCTCGATCTATCTGAGCGCCGAATGAAGAACCATTTCTGGGCTTACTTGCCGATCTGGGCTTC
        GACGCTGTTATTGCCGTTCATGTACTTCGGCTTCGCGTGGTCGGTAGCAAATAACGATCTGACCGGATTGCAA
        ATGGCAGCCGGTGTGCTGGGGTTGGCATGGCTTTCGGTTGTGCCTGGCGTTCCTGCAACCCACGAGTTGTATC
        ACTCGCGCGGGCGGTTGGCTCGCTTTGTTGGGCGCTATGGCCAGATCGTATTTCTCGATGTCATGCGCATGGA
        AGTGCACGTAGTCGGCCACCACCGCGACGTGGGGACGGCTGAGGACATTGATACTGCGGCACGCGGTATGAAT
        CTGTACAGATTCGTCGTTCGTGCAGTGGTGGAGTCCGTCCGCTGGGAGCTGAAGCTGGATGCAGACAATCTGG
        AGAAACGCGGTTACGGTCGCTATTCGATCCGTCACTCGGTCTGGCGCGCGCTGCTGGCCATTGTAGTGTTCCT
        GGGGTGCATCTATGCCATTGGCGGTGGGATGGCGGTGGGTCTTTGTTCGATCTCGATGATCATCGCTCGTTTT
        TGGGTTGAGGCATTCAACTACTACCAGCATTACGGCCAAGTCCGTTTGGTCGGCATGCCCATCGAAAAGCGGC
        ACGTCTGGAATCACTTTGGAACGCTCTCACGCTTGT – 3'</p>

      <h2>Construction of the Insert DNA</h2>
      <div class="dividerbar"></div>
      <p>UHA and DHA sequences were joined in silico to form a continuous segment.
          <br><br>
        Restriction sites for XmaI and PstI were added at the ends to allow the insert to ligate with the plasmid backbone.
          <br><br>
        Primers were designed specific to this segment with the following parameters:</p>
        <ul>
          <li>Length: ~35 nucleotides</li>
          <li>GC content: 50–60%</li>
          <li>Melting temperature (T<sub>m</sub>)</li> 
        </ul>

        <div class="image-and-caption">
        <img src="m5.png">
        <p class="caption">Fig : pst1 Restriction Site</p>
        </div>

        <div class="image-and-caption">
        <img src="m6.png">
        <p class="caption">Fig : Xma1 Restriction Site</p>
        </div>

        <div class="image-and-caption">
        <img src="m7.png">
        <p class="caption">Forward primer design : CCCGGGCTGCCATGAGACTGAAAGA</p>
        </div>

        <div class="image-and-caption">
        <img src="m8.png">
        <p class="caption">Reverse Primer Design : GGAACGCTCTCACGCTTGTCTGCAG</p>
        </div>

      <h2>Future: Flux Balance Analysis (FBA)</h2>
      <div class="dividerbar"></div>
      <p>Flux balance analysis (FBA) is a widely used approach for studying biochemical 
        networks, particularly genome-scale metabolic network reconstructions that contain 
        all known metabolic reactions in an organism and the genes that encode each enzyme
        (What Is Flux Balance Analysis?, n.d.). <br>
        It is a mathematical modelling technique to simulate biochemical reactions as a 
        linear equation system in silico under the steady-state assumption. FBA allows us 
        to analyse the pathways occurring in organisms, identify which reactions must be 
        regulated for the end product, and suggest further optimisations concerning growth 
        conditions and yield estimates.</p>
      <h4>Importance in our project</h4>
      <p>To enable rational metabolic engineering and reduce experimental trial-and-error, 
        our project will incorporates Flux Balance Analysis (FBA) as a computational tool. 
        FBA is a constraint-based modelling approach that predicts intracellular metabolic 
        flux distributions under defined physiological constraints, without requiring enzyme 
        kinetic parameters. <br>
        In our project, FBA will be used to evaluate the feasibility of redirecting carbon 
        flux in Pseudomonas putida toward the production of p-cumic aldehyde from p-cymene, 
        to understand the effect of the gene knockout, and to guide the selection of genetic 
        interventions prior to wet-lab implementation. <br>
        A genome-scale metabolic model of Pseudomonas putida F1 will be used to simulate 
        metabolic flux distributions under defined constraints. The engineered p-cymene 
        to p-cumic aldehyde pathway will be integrated into the model, and FBA will be 
        performed to evaluate pathway feasibility, identify metabolic bottlenecks, 
        and predict trade-offs between growth and product formation.</p>

      <h4>GSMM</h4>
      <p>There is no published and well cited genome-scale metabolic model (GEM) of <em>Pseudomonas putida F1</em>
        that can be used as the modelling framework. Hence, we will be using existing models developed for 
        closely related strains (e.g., KT2440) and refine them to represent the metabolic capabilities 
        specific to P. putida F1. The p-cymene catabolic pathway and engineered steps leading to p-cumic 
        aldehyde formation and accumulation will then be incorporated into the model using stoichiometric 
        information from literature and metabolic databases to get our final objectives. <br>
        The model assumes steady-state conditions, enforcing mass balance for all intracellular metabolites 
        (S.v = 0), and applies biologically realistic constraints on reaction directionality and flux bounds. <br>
        The objective (knockout effect in our case) is written as: <br>
        </p>
        <p class="formula"> Z=c<sup>T</sup>v</p>
        <p>Where,</p>
        <ul>
          <li>v = all reaction fluxes</li>
          <li>c = a vector for the flux that has an effect on our objective</li>
          <li>Z = the numerical score of the effectiveness of the solution</li>
        </ul>

        <h4>General Flow of action</h4>
        <div class="image-and-caption">
        <img src="m9.jpeg">
        <p class="caption">Fig 1: General flow of action of FBA</p>
        </div>

        <h4>Proposed Protocol: Flux Balance Analysis</h4>
        <ul>
          <li><u>Step 1: Model Selection</u><br>
          A curated genome-scale metabolic model of P. putida F1 will be selected (formulated if 
          there is no published/well cited model for our strain) and validated for growth under 
          minimal medium conditions.</li>
          <li><u>Step 2: Pathway Integration</u><br>
          Reactions constituting the p-cymene to p-cumic aldehyde pathway will be incorporated 
          into the model with stoichiometric and thermodynamic constraints derived from literature 
          and databases (KEGG, MetaCyc).</li>
          <li><u>Step 3: Constraint Definition</u><br>
          Physiological constraints such as substrate uptake rates, reaction reversibility, gene 
          deletions, and reaction fluxes will be applied.</li>
          <li><u>Step 4: Objective Formulation</u><br>
          Multiple objective functions will be tested, including biomass maximization, aldehyde 
          production maximization, and multi-objective (Pareto) optimization. <br>
          FBA will be performed using biomass maximization and aldehyde production maximization.</li>
          <li><u>Step 5: Decision-Making</u><br>
          Flux distributions will be analyzed to identify bottlenecks and optimal engineering strategies.</li>
          <li><u>Step 6: Experimental Translation</u><br>
          FBA-supported strategies will guide gene knockouts, overexpression targets, and pathway balancing 
          prior to wet-lab implementation.</li>
        </ul>

        <h4>Applications of FBA in Our Project</h4>
        <ol>
          <li><u>Pathway Feasibility Assessment</u><br>
          FBA is used to verify whether the engineered pathway can carry non-zero flux toward p-cumic aldehyde 
          production under physiological constraints. This allows us to rule out infeasible pathway designs 
          before experimental implementation.</li>
          <li><u>Identification of Metabolic Bottlenecks</u><br>
          By analyzing predicted flux distributions and performing flux variability analysis (FVA), we identify 
          reactions that limit carbon flow toward the target product. These reactions are prioritized as 
          candidates for overexpression or pathway balancing.</li>
          <li><u> Evaluation of Gene Knockouts</u><br>
          FBA simulations are used to assess the impact of knocking out competing reactions, such as aldehyde 
          dehydrogenases that convert p-cumic aldehyde to downstream metabolites. This helps ensure that 
          knockouts do not completely block growth or destabilize central metabolism (What Is Flux Balance 
          Analysis?, n.d.).</li>
        </ol>

        <h4>Limitations</h4>
        <ul>
          <li>It cannot predict metabolite concentrations as it does not use kinetic parameters.</li>
          <li>It can only determine the fluxes at steady state.</li>
          <li>The reliability of this models predictions depends on the accuracy of the stoichiometric data.</li>
        </ul>

      <h2>References</h2>
      <div class="dividerbar"></div>
      <ol>
        <li><a href="https://en.vectorbuilder.com/tool/gc-content-calculator.html">GC Content Calculator | VectorBuilder</a></li>
        <li><a href="https://www.neb.com/en/products/r0140-psti?srsltid=AfmBOop5h8nbatBPoXMHjdzSdOCw9BNa9uk0VPAhNIFVu2yQe9EW8m5w">PstI | NEB</a></li>
        <li><a href="https://www.neb.com/en/products/r0180-xmai?srsltid=AfmBOorQmYOQpW5N06osUuJjbh4YaOueoCRlOqLGk3Gaxz1slQKCeJMg">XmaI | NEB</a></li>
        <li><a href="https://doi.org/10.1038/nbt.1614">Orth, J. D., Thiele, I., & Palsson, B. Ø. (2010). What is flux balance analysis? Nature Biotechnology, 28(3), 245–248.</a></li>
      </ol>
    </main>
  </div>
  <footer>
  <video
    id="footerVideo"
    preload="metadata"
    playsinline
    muted
  >
    <source src="../../footervid.mp4" type="video/mp4">
  </video>
  <p>© 2025 Green Team <br>
       <em>for the taskphase of</em> <br>
       Manipal BioMachines. <br>
       <br>
      View our <a href="https://github.com/poseidon-boi/mbm-taskphase-green-team"><em>source code</em></a> <br><br>
      <em class="psst">psst- click the bottle!</em>
      
    </p>
</footer>
    <script src="../../script.js"></script>
</body>
</html>