<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Cormorant+Garamond:ital,wght@0,300..700;1,300..700&family=Noto+Serif:ital,wght@0,100..900;1,100..900&family=VT323&display=swap" 
    rel="stylesheet">
    <link href="https://fonts.googleapis.com/css2?family=MonteCarlo&display=swap" 
    rel="stylesheet">
    <link href="https://fonts.googleapis.com/css2?family=Oranienbaum&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="../../pinkAndYellow.css">
    <link rel="icon" type="image/png" href="../../favicon.png">
    <title>Model: Green Team</title>
</head>
<body>
  <div id="loader">
    <video autoplay muted loop playsinline preload="auto">
      <source src="../../load.mov" type="video/mp4">
    </video>
    <h1>LOADING...</h1>
  </div>
    <nav>
        <div class="logo"><a href="../../index.html">green team</a></div>
        <div class="nav-items">

          <div class="dropdown">
            <button class="dropbtn" id="projectBtn">Project</button>
            <div class="dropdown-content" id="projectMenu">
              <a href="../project/description.html">Description</a>
              <a href="../project/design.html">Design</a>
              <a href="../project/proofOfConcept.html">Proof of Concept</a>
              <a href="../project/results.html">Results</a>
              <a href="../project/implementation.html">Implementation</a>
              <a href="../project/safety.html">Safety</a>
              <a href="../project/contribution.html">Contribution</a>
            </div>
          </div>

          <div class="dropdown">
            <button class="dropbtn" id="wetLabBtn">Wet Lab</button>
            <div class="dropdown-content" id="wetlabMenu">
              <a href="../wetlab/engineering.html">Engineering</a>
              <a href="../wetlab/experiments.html">Experiments</a>
              <a href="../wetlab/notebook.html">Notebook</a>
              <a href="../wetlab/measurements.html">Measurement</a>
              <a href="../wetlab/labSafety.html">Lab Safety</a>
              <a href="../wetlab/protocols.html">Protocols</a>
            </div>
          </div>

          <div class="dropdown">
            <button class="dropbtn" id="dryLabBtn">Dry Lab</button>
            <div class="dropdown-content" id="drylabMenu">
              <a href="model.html">Model</a>
              <a href="parts.html">Parts</a>
              <a href="software.html">Software</a>
            </div>
          </div>

          <div class="dropdown">
            <button class="dropbtn" id="engagementBtn">Engagement</button>
            <div class="dropdown-content" id="engagementMenu">
              <a href="../engagement/hp.html">Human Practices</a>
              <a href="../engagement/ihp.html">Integrated Human Practices</a>
              <a href="../engagement/collaborations.html">Collaborations</a>
              <a href="../engagement/sustainability.html">Sustainability</a>
              <a href="../engagement/education.html">Education</a>
            </div>
          </div>

          <div class="dropdown">
            <button class="dropbtn" id="teamBtn">Team</button>
            <div class="dropdown-content" id="teamMenu">
              <a href="../team/members.html">Members</a>
              <a href="../team/attributions.html">Attributions</a>
              <a href="../team/medals.html">Medals</a>
              <a href="../team/sponsors.html">Sponsors</a>
            </div>
          </div>

        </div>
      </nav>
      <div class="head fade-in">
        <h1 class="title">Modelling</h1>
      </div>
  <div class="page">
    <aside id="toc-container">
      <button id="toc-toggle">Contents</button>
      <nav id="toc"></nav>
    </aside>

    <main id="content">
      <h2>Whole-Genome BLAST Analysis Against the Target Aldehyde Gene</h2>
      <div class="dividerbar"></div>
      <p>We performed a whole-genome BLAST analysis of <em>Pseudomonas putida F1</em> against our target 
        aldehyde-related gene (cymC) to identify homologous sequences and confirm the presence.
        <br><br>
        Before designing a gene knockout strategy, it was essential to:</p>
        <ul>
          <li>Confirm that cymC is present as a single, well-defined locus in the genome</li>
          <li>Check for paralogous or homologous genes that could compensate for cymC deletion</li>
          <li>Ensure that targeting cymC would not unintentionally affect other aldehyde-metabolising enzymes</li>
          <li>Verify that the gene sequence used for the Upstream Homologous Arm and the Downstream Homologous Arm design is accurate</li>
        </ul>

        <p>A BLASTn search was performed using the <em>Pseudomonas putida F1</em> genome as the subject.</p>
        <div class="image-and-caption">
        <img src="m1.png">
        <p class="caption">Fig 1: BLAST analysis of the CymC protein sequence against the whole genome of <em>P. putida F1</em></p>
      </div>

      <p>Inference : BLAST analysis showed a perfect match between the query gene and the <em>Pseudomonas putida F1</em> genome, with 100% identity 
        and an E-value of 0, indicating a highly significant alignment. </p>

      <h2>Aldehyde Dehydrogenase Specificity Analysis</h2>
      <div class="dividerbar"></div>
      <p>We performed a comparative BLAST analysis to evaluate whether other aldehyde dehydrogenase 
        enzymes present in <em>Pseudomonas putida F1</em> could potentially interfere with or compensate for 
        the function of our target aldehyde-related gene (cymC).A. putida contains multiple aldehyde 
        dehydrogenases involved in diverse metabolic pathways. <br>Before proceeding with a gene knockout 
        strategy, it was important to assess:</p>
        <ul>
          <li>Potential functional redundancy</li>
          <li>Possible off-target aldehyde metabolism</li>
          <li>The likelihood of other enzymes compensating for the loss of cymC</li>
        </ul>

        <p>Reviewed aldehyde dehydrogenase proteins from P. putida F1 were retrieved from UniProt (Swiss-Prot). 
          A total of six aldehyde dehydrogenase candidates were identified. The protein sequences were individually 
          subjected to BLAST analysis with CymC. </p>

      <div class="image-and-caption">
        <img src="m2.png">
        <p class="caption">Fig 2 : BLAST analysis of <em>p.putida</em> versus structural homologous Aldehyde Dehydrogenase.</p>
      </div>

      <p>The BLAST results show that while <em>P. putida</em> has other aldehyde dehydrogenases, the closest hits (betB and astD) have only moderate to low sequence similarity to cymC. 
<br><br>Active Site Conservation Analysis of cymC
We investigated the active site of cymC, which is reported as GLU265 on UniProt. <br><br>
To find homologous sequences, we performed a BLAST search of cymC against <em>Pseudomonas putida</em> F1 aldehyde dehydrogenase sequences.
<br><br>From this, two sequences with the highest alignment scores were selected. <br><br>
We then performed a multiple sequence alignment using Clustal Omega on cymC and the two BLAST hits to check for conserved residues.</p>
     
        <div class="image-and-caption">
        <img src="m10.jpeg">
        <p class="caption">Fig 3 : Clustal Omega Alignment of cymC, betB and astD genes.</p>
      </div>

      <p>The Clustal Omega alignment showed that the 266th position in cymC and the corresponding positions in both hits is glutamate (E).
<br><br>

This indicates that this region is conserved among closely aligned sequences, which can be used to filter out non-specific homologs in further studies.
<br><br>
We then used PyMOL to visualise the three protein structures and find the RMSD values for each pair. The following are the results.
<br><br>
 Global structural similarity <br>
CymC vs BetB <br>
RMSD = 0.813 Å = Very strong structural homology</p>

    <h2>Global structural similarity</h2>
    <div class="dividerbar"></div>
    <p>CymC vs BetB: RMSD = 0.813 Å = Very strong structural homology</p>
    <div class="image-and-caption">
        <img src="m11.jpeg">
        <p class="caption">Fig 4 : RSMD value for the superimposition of CymC and betB</p>
      </div>

    <p>CymC vs AstD: RMSD = 1.140 Å = Strong structural homology</p>
    <div class="image-and-caption">
        <img src="m12.jpeg">
        <p class="caption">Fig 5 : RSMD value for the superimposition of CymC and astD</p>
      </div>

    <h4>Interpretation</h4>

<ul>
  <li>These RMSD values indicate strong global fold conservation.</li>
  <li>BetB is more structurally similar to CymC than AstD.</li>
</ul>

<p>
  From this, we can conclude that BetB and AstD are bona fide ALDH-like enzymes
  that should be used for testing potential off-target substrate binding.
</p>


<h4>Active Site Comparison</h4>

<p>
  The positions of the active sites for CymC (O33455), AstD (A5W0D5), and BetB
  (A5WA96) were obtained from the UniProt database.
</p>

<p>
  CymC had only one active-site residue annotated (E265). Therefore, the
  selection area was expanded to define the full active pocket, which was then
  superimposed with the active-site regions of AstD and BetB for comparison.
</p>

<div class="image-and-caption">
        <img src="m13.jpeg">
        <p class="caption">Fig 6 : PyMOL log for active site similarity analysis</p>
      </div>

    <h4>Sensitivity to p-Cumic Aldehyde</h4>
<p>AstD and betB have a similar pocket structure to CymC, but they're not non-specific substitutes for p-cumic oxidation, with substrate pockets preferring aliphatic/amino groups over large aromatic groups, such as p-isopropylbenzaldehyde. In conclusion, AstD and betB in Pseudomonas putida F1 are unlikely to significantly bind or oxidize p-cumic aldehyde due to their specialized functions and substrate specificities, despite pocket similarities.</p>


    

      <h2>Identification of Upstream and Downstream Homology Arms</h2>
      <div class="dividerbar"></div>
      <p>We identified the regions flanking the target gene (cymC) in the <em>P. putida F1</em> genome to design the 
        upstream and downstream homology arms (UHA and DHA) required for homologous recombination.
      <br><br>
        Using the BLAST results from the genome vs. cymC gene, we determined the exact genomic coordinates of the 
        target gene.
        <br><br>     
        The orientation (plus/minus strand) was noted to correctly identify upstream and downstream regions.
        <br><br>
        Sequences upstream (UHA) and downstream (DHA) of the gene were extracted. Reverse complement sequences 
        were converted to 5′ → 3′ orientation to prepare them for primer design and cloning.
      </p> 
        <div class="image-and-caption">
        <img src="m3.jpeg">
        <p class="caption">Fig 7 : BLAST sequence alignment of <em>p.putida</em> versus cymC gene. </p>
      </div>

      <div class="image-and-caption">
        <img src="m4.jpeg">
        <p class="caption">Fig 8: BLAST sequence alignment of <em>p.putida</em> versus cymC gene (contd.). </p>
      </div>
      
      <p>UHA sequence : <br></p>
      <p class="dna">5' - CTGCCATGAGACTGAAAGACAAAGTCGCAATCGTGACTGGTGCGGCAACCGGGA
        TTGGCAACGCTATCGTGCGCTCTTACCTTGCTGAGGGTGCGAAGGTGGTCATTGCTGATGTGAAAGGTGCTG
        AGGCAGCCGCGGCGGAACTGGGTGAAGACCTAGCGTTGGGTGTCTTTGCCGACGTTTCGGATCCTGATAGCA
        CCAAGCAGATGGCAAAGGCAGCACTGGATCGCTTCGGCAAGATCGACGTGCTCGTCAACAATGCCGGTATCT
        TCACCGGTCTCAACTATGTCCCGATGGAATCGATCTCGGTTGCGGATTGGGACAAGCTCTACAGTGTCAATG
        TCAAGGGGCCCTGGCTGTGCGCCAGTGCGGTGTCGGCCGCGATGCGCGAGGGTGGTGGTGGCAAGATCATCA
        ACATCGCCTCAGTAATCGCTCACATCGGAGCGCCCTTCATGTTGCACTACGTTTCCAGCAAGGGCGCTGTTG
        CGGCGATGACACGTGCCATGGCGCGCGAATTTGCGACGACCAAGGCAGGGATCAGCGTCAACTCCATCTCTC
        CCGGCTACACCCATTCGGAAAATGCGCTCGCAAACGCTCAGCAGCATGAGCAGTTCGAGGGCGTTTCTGCGT
        CCATGCGGGCAATCGATCGACCGCAAGTGCCGGCAGATATTGCGGGAGTGGCGCTTTGGTTGGCTAGCGACG
        AAGCCTCCTACGTAAACGGGCAGAACATTGTGGTGGATGGCGGCATTTACATGAGCCTCTGAGCGTCATTCG
        CCACGACATTCATCCTTAGTTGTGATTT – 3' </p>

      <p>DHA sequence : <br></p>
      <p class="dna">5’ - ACGAAATCCTGCCCTCGCGTGCGAGGACGTCACGTCCGCTTAAGGAGACCGATC
        <strong>ATG</strong>TGGGAATACATCAAGTACTACTTCGCACCCCTGGTACAAGTCTTTGCGATTC
        TGGGGTTCTATTGGGGTGGTCATTACACTTGGATTGCAATTGGGGCTTTTCCGGCCATTGCCATCCTTGACA
        TGCTGCTGCCGCTCGATCTATCTGAGCGCCGAATGAAGAACCATTTCTGGGCTTACTTGCCGATCTGGGCTTC
        GACGCTGTTATTGCCGTTCATGTACTTCGGCTTCGCGTGGTCGGTAGCAAATAACGATCTGACCGGATTGCAA
        ATGGCAGCCGGTGTGCTGGGGTTGGCATGGCTTTCGGTTGTGCCTGGCGTTCCTGCAACCCACGAGTTGTATC
        ACTCGCGCGGGCGGTTGGCTCGCTTTGTTGGGCGCTATGGCCAGATCGTATTTCTCGATGTCATGCGCATGGA
        AGTGCACGTAGTCGGCCACCACCGCGACGTGGGGACGGCTGAGGACATTGATACTGCGGCACGCGGTATGAAT
        CTGTACAGATTCGTCGTTCGTGCAGTGGTGGAGTCCGTCCGCTGGGAGCTGAAGCTGGATGCAGACAATCTGG
        AGAAACGCGGTTACGGTCGCTATTCGATCCGTCACTCGGTCTGGCGCGCGCTGCTGGCCATTGTAGTGTTCCT
        GGGGTGCATCTATGCCATTGGCGGTGGGATGGCGGTGGGTCTTTGTTCGATCTCGATGATCATCGCTCGTTTT
        TGGGTTGAGGCATTCAACTACTACCAGCATTACGGCCAAGTCCGTTTGGTCGGCATGCCCATCGAAAAGCGGC
        ACGTCTGGAATCACTTTGGAACGCTCTCACGCTTGT – 3'</p>

      <h2>Construction of the Insert DNA</h2>
      <div class="dividerbar"></div>
      <p>UHA and DHA sequences were joined in silico to form a continuous segment.
          <br><br>
        Restriction sites for XmaI and PstI were added at the ends to allow the insert to ligate with the plasmid backbone.
          <br><br>
        Primers were designed specific to this segment with the following parameters:</p>
        <ul>
          <li>Length: ~35 nucleotides</li>
          <li>GC content: 50–60%</li>
          <li>Melting temperature (T<sub>m</sub>)</li> 
        </ul>

        <div class="image-and-caption">
        <img src="m5.png">
        <p class="caption">Fig 9 : pst1 Restriction Site</p>
        </div>

        <div class="image-and-caption">
        <img src="m6.png">
        <p class="caption">Fig 10: Xma1 Restriction Site</p>
        </div>

        <div class="image-and-caption">
        <img src="m7.png">
        <p class="caption">Fig 11: Forward primer design : CCCGGGCTGCCATGAGACTGAAAGA</p>
        </div>

        <div class="image-and-caption">
        <img src="m8.png">
        <p class="caption">Fig 12: Reverse Primer Design : GGAACGCTCTCACGCTTGTCTGCAG</p>
        </div>

    

      <h2>References</h2>
      <div class="dividerbar"></div>
      <ol>
        <li><a href="https://en.vectorbuilder.com/tool/gc-content-calculator.html">GC Content Calculator | VectorBuilder</a></li>
        <li><a href="https://www.neb.com/en/products/r0140-psti?srsltid=AfmBOop5h8nbatBPoXMHjdzSdOCw9BNa9uk0VPAhNIFVu2yQe9EW8m5w">PstI | NEB</a></li>
        <li><a href="https://www.neb.com/en/products/r0180-xmai?srsltid=AfmBOorQmYOQpW5N06osUuJjbh4YaOueoCRlOqLGk3Gaxz1slQKCeJMg">XmaI | NEB</a></li>
        <li><a href="https://doi.org/10.1038/nbt.1614">Orth, J. D., Thiele, I., & Palsson, B. Ø. (2010). What is flux balance analysis? Nature Biotechnology, 28(3), 245–248.</a></li>
        <li>
    <em>Pseudomonas putida</em> F1, complete genome sequence:
    <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_009512.1?report=fasta" target="_blank">
      NC_009512.1 (FASTA)
    </a>
  </li>

  <li>
    <em>Pseudomonas putida</em> <em>p-</em>cumic aldehyde dehydrogenase (ENA):
    <a href="https://www.ebi.ac.uk/ena/browser/view/AAB62298.1" target="_blank">
      AAB62298.1
    </a>
  </li>

  <li>
    <em>p-</em>cumic aldehyde dehydrogenase [<em>Pseudomonas putida</em>] (NCBI Protein):
    <a href="https://www.ncbi.nlm.nih.gov/protein/AAB62298.1?report=fasta" target="_blank">
      AAB62298.1 (FASTA)
    </a>
  </li>

  <li>
    UniProt entry for BetB (A5WA96):
    <a href="https://www.uniprot.org/uniprotkb/A5WA96/entry#function" target="_blank">
      UniProt A5WA96
    </a>
  </li>

  <li>
    UniProt entry for AstD (A5W0D5):
    <a href="https://www.uniprot.org/uniprotkb/A5W0D5/entry#sequences" target="_blank">
      UniProt A5W0D5
    </a>
  </li>
      </ol>
    </main>
  </div>
  <footer>
  <video
    id="footerVideo"
    preload="metadata"
    playsinline
    muted
  >
    <source src="../../footervid.mp4" type="video/mp4">
  </video>
  <p>© 2025 Green Team <br>
       <em>for the taskphase of</em> <br>
       Manipal BioMachines. <br>
       <br>
      View our <a href="https://github.com/poseidon-boi/mbm-taskphase-green-team"><em>source code</em></a> <br><br>
      <em class="psst">psst- click the bottle!</em>
      
    </p>
</footer>
    <script src="../../script.js"></script>
</body>
</html>